go/analyzer/staticcheck/util/util.go

```diff
@@ -0,0 +1,47 @@
+package util
+
+import (
+	"fmt"
+	"sort"
+
+	"golang.org/x/tools/go/analysis"
+	"honnef.co/go/tools/analysis/lint"
+	"honnef.co/go/tools/quickfix"
+	"honnef.co/go/tools/simple"
+	"honnef.co/go/tools/staticcheck"
+	"honnef.co/go/tools/stylecheck"
+)
+
+var StaticcheckAnalyzers = [][]*lint.Analyzer{
+	quickfix.Analyzers,
+	simple.Analyzers,
+	staticcheck.Analyzers,
+	stylecheck.Analyzers,
+}
+
+var analyzerMap, AnalyzerSlice = func() (map[string]*analysis.Analyzer, []*analysis.Analyzer) {
```

Comment:
why is the slice public?

Reply:
Will remove slice entirely next iteration